test/helper/resource.go

```diff
@@ -765,3 +786,29 @@ func NewPodDisruptionBudget(namespace, name string, maxUnAvailable intstr.IntOrS
 		},
 	}
 }
+
+// NewWork will build a new Work object.
+func NewWork(workName, workNs string, raw []byte, deletionTimestampIsZero ...bool) *workv1alpha1.Work {
```

Comment:
`deletionTimestampIsZero` seems unused, even if reserved, `deletionTimestampIsZero bool` is enough.

Reply:
Done.